web/src/search/input/QueryInput.tsx

```diff
@@ -396,7 +398,7 @@ export class QueryInput extends React.Component<Props, State> {
                                         {this.state.loadingSuggestions && (
                                             <li className="suggestion suggestion--selected">
                                                 <LoadingSpinner className="icon-inline" />
-                                                <div className="suggestion__description">fetching suggestions...</div>
+                                                <div className="suggestion__description">LOADING</div>
```

Comment:
```suggestion
                                                <div className="suggestion__description">Loading</div>
```

Reply:
I set it capitalized based on this [comment](https://github.com/sourcegraph/sourcegraph/pull/6662#discussion_r347907186), maybe I understood it wrong?

Reply:
I think he means capitalized as in, the first letter should be capitalized, not the whole thing. See other places we do loading indicators: https://sourcegraph.com/search?q=r:sourcegraph/sourcegraph%24+Loading...+case:yes+f:%5C.tsx&patternType=literal.

You actually might want to add an ellipses too: 

```suggestion
                                                <div className="suggestion__description">Loading...</div>
```